README.md

```diff
@@ -207,31 +162,125 @@ Users want to determine several types of the formatting width as following
+  * `years` (`String`): The style to be used for formatting years. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `yearsDisplay` (`String`): When to display years. `"auto"` means that years won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `years` is `undefined` and `"always"` otherwise.
+  * `months` (`String`): The style to be used for formatting months. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `monthsDisplay` (`String`): When to display months. `"auto"` means that months won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `months` is `undefined` and `"always"` otherwise.
+  * `weeks` (`String`): The style to be used for formatting weeks. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `weeksDisplay` (`String`): When to display weeks. `"auto"` means that weeks won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `weeks` is `undefined` and `"always"` otherwise.
+  * `days` (`String`): The style to be used for formatting days. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `daysDisplay` (`String`): When to display days. `"auto"` means that days won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `days` is `undefined` and `"always"` otherwise.
+  * `hours` (`String`): The style to be used for formatting hours. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `hoursDisplay` (`String`): When to display hours. `"auto"` means that hours won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `hours` is `undefined` and `"always"` otherwise.
+  * `minutes` (`String`): The style to be used for formatting minutes. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `minutesDisplay` (`String`): When to display minutes. `"auto"` means that minutes won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `minutes` is `undefined` and `"always"` otherwise.
+  * `seconds` (`String`): The style to be used for formatting seconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `secondsDisplay` (`String`): When to display seconds. `"auto"` means that seconds won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `seconds` is `undefined` and `"always"` otherwise.
+  * `milliseconds` (`String`): The style to be used for formatting milliseconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
```

Comment:
I'd expect some discussion here (and in the microseconds and nanoseconds docs below) about:
* when ms/μs/ns values will be displayed as decimals after `seconds` (or after milliseconds or microseconds for smaller units) and when they'll be  displayed as standalone values like "123 ms".
* how `fractionalDigits` affects display of ms/μs/ns units
* whether trailing zeroes are shown or hidden by default